src/main/java/tr/com/orioninc/laborant/service/LabService.java

```diff
@@ -65,11 +67,9 @@ public String getALlLabsStatus(){
         }
         else{
             String response = "";
-            String outputString = "";
-            for (Lab currentLab: allLabs
-                 )
-            {
-                outputString+= currentLab.getLabName() +   "    "+ "Host: "+ currentLab.getHost()  +  "      " ;
+            StringBuilder outputString = new StringBuilder();
+            for (Lab currentLab: allLabs) {
+                outputString.append("Lab: ").append(currentLab.getLabName()).append("  ").append("Host: ").append(currentLab.getHost()).append("----");
```

Comment:
Override toString method for Lab class and simply use `currentLab.toString()`

Reply:
done